components/assign-dropdown-select.tsx

```diff
@@ -0,0 +1,62 @@
+import Select from 'react-select';
+import { Button } from './button';
+import './dropdown-select.css';
+import { useState } from 'react';
+import { ASSIGN_USERS } from '@/utils/constants/common-constants';
+import ShowModal from './assign-modal';
```

Comment:
I would like to rename this modal name to `TransferConfirmationModal` instead of `ShowModal`.